MapboxNavigation/VisualInstructionComponent.swift

```diff
@@ -0,0 +1,55 @@
+import UIKit
+import SDWebImage
+import MapboxDirections
+
+extension VisualInstructionComponent {
+    
+    static let shieldURLCache = NSCache<NSString, NSURL>()
```

Comment:
`shieldURLCache` was used to cache the `imageURL` from the Wikimedia response. We don't need this cache anymore since the URL is always present in the instructions component if it exists.